Android_Development_with_Kotlin/07. View Groups/07.2 Relative Layout in Kotlin.md

```diff
@@ -0,0 +1,141 @@
+|layout_toEndOf	|It accepts another sibling view id and places the view to end of the specified view id.|
+|layout_above	|It accepts another sibling view id and places the view above the specified view id.|
+|layout_below	|It accepts another sibling view id and places the view below the specified view id.|
+
+## <div align="center">🤔How to declare RelativeLayout in XML file?</div>
+First of all, we should declare the RelativeLayout in layout file using the below code.
+
+```
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="10dp"
+    android:paddingRight="10dp">
+  
+    // Add other view or ViewGroup here
+</RelativeLayout>
+```
+
+## <div align="center">📃RelativeLayout in activity_main.xml file</div>
+Following is the code for RelativeLayout in xml file.
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="10dp"
+    android:paddingRight="10dp">
+     
+    <TextView
+        android:id="@+id/textView1"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentLeft="true"
+        android:text="First name:"
+        android:layout_marginTop="20dp"
+        android:textSize="20dp"/>
+  
+    <EditText
+        android:id="@+id/editText1"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_toRightOf="@id/textView1"
+        android:layout_marginTop="8dp"/>
+      
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentLeft="true"
+        android:layout_below="@+id/textView1"
+        android:layout_marginTop="10dp"
+        android:text="Last name:"
+        android:textSize="20dp"/>
+  
+    <EditText
+        android:id="@+id/editText2"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        android:layout_toRightOf="@id/textView2"
+        android:layout_marginTop="45dp"/>
+  
+    <Button
+        android:id="@+id/btn4"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        android:layout_below="@id/textView2"
+        android:layout_marginTop="20dp"
+        android:text="Submit" />
+      
+</RelativeLayout>
+```
+
+## <div align="center">🧠MainActivity.kt file</div>
+When we have created layout, we need to load the XML layout resource from our activity onCreate() callback method and access the UI element form the XML using findViewById.
+
+
+<pre><code>
+package com.geeksforgeeks.myfirstKotlinapp

```

Comment:
rename the package name so that we don't get copyright issues.

Mention the reference link in the bottom